aiida_restapi/models.py

```diff
@@ -62,6 +67,42 @@ def get_projectable_properties(cls) -> List[str]:
         """Return projectable properties."""
         return list(cls.schema()["properties"].keys())
 
+    @staticmethod
+    def get_download_formats(full_type: str | None = None) -> dict:
```

Comment:
That is consistent with the name in aiida-core
```suggestion
    def get_all_download_formats(full_type: str | None = None) -> dict:
```

Reply:
Makes more sense to be moved to some utils/common.py